diff --git a/node_modules/expo-widgets/ios/ExpoWidgets.podspec b/node_modules/expo-widgets/ios/ExpoWidgets.podspec
index 1d2b971..f8c945a 100644
--- a/node_modules/expo-widgets/ios/ExpoWidgets.podspec
+++ b/node_modules/expo-widgets/ios/ExpoWidgets.podspec
@@ -42,7 +42,8 @@ Pod::Spec.new do |s|
     :script => %Q{
       echo "Preparing ExpoWidgets.bundle..."
       source="#{__dir__}/../bundle/build/ExpoWidgets.bundle"
-      dest="${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/ExpoWidgets.bundle"
+      dest="${TARGET_BUILD_DIR}/ExpoWidgets.bundle/ExpoWidgets.bundle"
+      mkdir -p "${TARGET_BUILD_DIR}/ExpoWidgets.bundle"
       echo "Copying ${source} to ${dest}"
       cp "${source}" "${dest}"
     },
@@ -56,7 +57,7 @@ Pod::Spec.new do |s|
   end
   s.script_phases = [build_bundle_script, copy_bundle_script]
   s.resource_bundles = {
-    'ExpoWidgets' => [],
+    'ExpoWidgets' => ["../bundle/build/ExpoWidgets.bundle"],
   }
 
 end
diff --git a/node_modules/expo-widgets/scripts/xcode-build-bundle.sh b/node_modules/expo-widgets/scripts/xcode-build-bundle.sh
index 7dd91dc..3ba7832 100644
--- a/node_modules/expo-widgets/scripts/xcode-build-bundle.sh
+++ b/node_modules/expo-widgets/scripts/xcode-build-bundle.sh
@@ -4,16 +4,6 @@ set -eo pipefail
 
 EXPO_WIDGETS_PACKAGE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
 
-# For classic main project build phases integration, will be no-op to prevent duplicated bundle creation.
-#
-# `$PROJECT_DIR` is passed by Xcode as the directory to the xcodeproj file.
-# in classic main project setup it is something like /path/to/app/ios
-# in new style pod project setup it is something like /path/to/app/ios/Pods
-PROJECT_DIR_BASENAME=$(basename $PROJECT_DIR)
-if [ "x$PROJECT_DIR_BASENAME" != "xPods" ]; then
-  exit 0
-fi
-
 # If PROJECT_ROOT is not specified, fallback to use Xcode PROJECT_DIR
 PROJECT_ROOT=${PROJECT_ROOT:-"$PROJECT_DIR/../.."}
 PROJECT_ROOT=${PROJECT_ROOT:-"$EXPO_WIDGETS_PACKAGE_DIR/../.."}
