diff --git a/node_modules/expo-widgets/ios/ExpoWidgets.podspec b/node_modules/expo-widgets/ios/ExpoWidgets.podspec
index 1d2b971..f8c945a 100644
--- a/node_modules/expo-widgets/ios/ExpoWidgets.podspec
+++ b/node_modules/expo-widgets/ios/ExpoWidgets.podspec
@@ -42,7 +42,8 @@ Pod::Spec.new do |s|
     :script => %Q{
       echo "Preparing ExpoWidgets.bundle..."
       source="#{__dir__}/../bundle/build/ExpoWidgets.bundle"
-      dest="${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/ExpoWidgets.bundle"
+      dest="${TARGET_BUILD_DIR}/ExpoWidgets.bundle/ExpoWidgets.bundle"
+      mkdir -p "${TARGET_BUILD_DIR}/ExpoWidgets.bundle"
       echo "Copying ${source} to ${dest}"
       cp "${source}" "${dest}"
     },
@@ -56,7 +57,7 @@ Pod::Spec.new do |s|
   end
   s.script_phases = [build_bundle_script, copy_bundle_script]
   s.resource_bundles = {
-    'ExpoWidgets' => [],
+    'ExpoWidgets' => ["../bundle/build/ExpoWidgets.bundle"],
   }
 
 end
